trail/upload/process_uploads/header_standardizer.py

```diff
@@ -268,6 +289,44 @@ def standardizeMetadata(self):
     def standardizeWcs(self, hdu=None):
         """Standardize WCS data a given header.
 
+        Parameters
+        ----------
+        hdu : `object` or `None`, optional
+            An Astropy image-like HDU unit. Useful when dealing with
+            mutli-extension fits files where metadata is in the PrimaryHDU but
+            the WCS and image data are stored in the extensions.
+
+        Returns
+        -------
+        standardizedWCS : `upload.models.Wcs`
+            Standardized WCS keys and values.
+
+        Raises
+        ------
+        StandardizeWcsException
+            There was a problem in standardizing the header
+
+        Notes
+        -----
+        Tries to read the WCS from the header.
+        if that does not work then gives it to astrometry.net to look for a solution for the WCS
+        """
+        try:
+            astropy_fail = False
+            try:
+                header, dimX, dimY = self._astropyWcsReader(hdu)
+            except ValueError and TypeError:
+                astropy_fail = True
+            if astropy_fail:
+                header, dimX, dimY = self._astrometryNetSolver(self.filepath)
+            wcs = models.Wcs(**self._computeStandardizedWcs(header, dimX, dimY))
+        except ValueError and RuntimeError and TypeError as err:
```

Comment:
Can you make this a tuple - I don't want to risk some weird behavior with `and`s.